packages/event-handler/src/types/rest.ts

```diff
@@ -241,6 +243,40 @@ type WebResponseToProxyResultOptions = {
   isBase64Encoded?: boolean;
 };
 
+/**
+ * Configuration for request validation
+ */
+type RequestValidationConfig<T = unknown> = {
```

Comment:
Seems odd to me that these are all optional. Is there a way to express through the type system that at least one of these should be present?